docker-compose.yml

```
@@ -38,4 +38,15 @@ services:
     command:
       /bin/bash -c "bash bin/*.sh"
     depends_on:
-      - dynamodb
\ No newline at end of file
+      - dynamodb
+  postgresql:
+    image: postgres:14
+    container_name: loggy-postgresql
+    tty: true
+    ports:
+      - 5432:5432
+    volumes:
+      - ./rdb_data:/var/lib/postgresql/data
+      - ./.docker/postgresql/scripts:/docker-entrypoint-initdb.d
+    env_file:
+      - .env
```

```suggestion
    env_file:
      - packages/backend/.env
```

yarn env:cp ていう spm scripts の定義を見てもらえると！